src/models/GitHubAuth.js

```diff
@@ -1,46 +1,25 @@
-'use strict'
-
 import keytar from 'keytar'
-import ElectronConfig from 'electron-config'
-
-const isTest = process.env.NODE_ENV === 'test'
-const storage = isTest ? new ElectronConfig({ name: 'config-test' }) : null
 
 const SERVICE = 'gh-notifications-snoozer'
 const ACCOUNT = 'github'
-const KEY = 'token'
 
-class GitHubAuth {
+module.exports = class GitHubAuth {
   static isAuthenticated() {
-    if (!isTest) {
-      return this.getToken() !== null
-    }
-    return storage.has(KEY)
```

Comment:
I had a pickle of a time trying to get keytar to work in tests on Travis, I think it was. Did you get that working?